metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/utils/SyslogUtilsTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.metron.parsers.utils;
+
+import org.apache.metron.parsers.ParseException;
+import org.junit.Test;
+
+import java.time.ZoneOffset;
+
+import static org.junit.Assert.*;
+
+public class SyslogUtilsTest {
+
+    @Test
+    public void testRfc3164Timestamp() {
+        String originalTimestamp = "Oct  9 13:42:11";
+        assertEquals(getParsedEpochMillis(originalTimestamp), 1476020531000L);
+    }
+
+    @Test
+    public void testCiscoTimestamp() {
+        String originalTimestamp = "Oct 09 2015 13:42:11";
+        assertEquals(getParsedEpochMillis(originalTimestamp), 1444398131000L);
+    }
+
+    @Test
+    public void testRfc5424TimestampUTC() {
+        String originalTimestamp = "2015-10-09T13:42:11.52Z";
+        assertEquals(getParsedEpochMillis(originalTimestamp), 1444398131520L);
+    }
+
+    @Test
+    public void testRfc5424TimestampWithOffset() {
+        String originalTimestamp = "2015-10-09T08:42:11.52-05:00";
+        assertEquals(getParsedEpochMillis(originalTimestamp), 1444398131520L);
+    }
+
+    private long getParsedEpochMillis(String originalTimestamp) {
+        try {
```

Comment:
There is no need to try/catch here.  Just have the method `throw ParseException`.  Any of your test methods can also `throw ParseException`.  This simplifies the logic and JUnit will fail the test if a ParseException is thrown.